docs/README.md

```diff
@@ -36,13 +39,51 @@ const Component: FunctionComponent = () => {
 }
 ```
 
+### withCssHandles
+
+```tsx
+import React, { Component } from 'react'
```

Comment:
make this example shorter, only showing basic functionality, not every cssHandles usage case